tests/test_multitask_prompt_tuning.py

```diff
@@ -246,6 +246,7 @@ def test_bf16_inference(self) -> None:
         _ = mpt.generate(input_ids=input_ids, task_ids=task_ids)
 
     def test_generate_text_with_random_init(self) -> None:
+        torch.manual_seed(0)
```

Comment:
Does this also happen here ?

Reply:
I had not encountered it before but after fixing the seed in the other test, this one failed on some CI runs. I expect it's the same error and it was just coincidence that I encountered this in the other test only.